corehq/apps/app_manager/models.py

```diff
@@ -2086,7 +2108,7 @@ def extended_build_validation(self, error_meta, xml_valid, validate_module=True)
                 errors.append(error)
 
         module = self.get_module()
-        if module.has_schedule and not (self.schedule and self.schedule.anchor):
+        if module.has_schedule and not self.schedule and getattr(self.schedule, 'anchor', False):
```

Comment:
Won't `getattr(self.schedule, 'anchor', False)` always be False since the anchor is now in SchedulePhase